rasgotransforms/rasgotransforms/transforms/funnel/funnel.sql

```diff
@@ -1,7 +1,7 @@
 {%- for col_name in stage_columns -%}
     SELECT
     '{{ col_name }}' AS LABEL
-    ,SUM({{ col_name }}) AS "Count"
+    ,SUM({{ col_name }}) AS LABEL_COUNT
```

Comment:
@szaghikian-rasgo is this new column alias going to break anything in the UI insights chart?

Reply:
My understanding is that in these transforms, the UI doesn't care explicitly about column names, but I'm double checking with @phillip-rasgo to be sure